FrontEnd/src/utils/validateEdrpou.js

```diff
@@ -0,0 +1,26 @@
+const EDRPOU_PATTERN = /^\d{8}$/;
+
+function validateEdrpou(edrpou) {
+    function calculateKey(weights, extra = 0) {
+        return weights.reduce((sum, weight, idx) => sum + (weight + extra) * parseInt(edrpou[idx], 10), 0) % 11;
+    }
+    if (EDRPOU_PATTERN.test(edrpou)) {
+        const weightsBase = (parseInt(edrpou, 10) > 30000000 && parseInt(edrpou, 10) < 60000000)
```

Comment:
Could you provide an explanation comment for these calculations?

Reply:
Comments explaining the validation have been added.